helm/charts/hpe-csi-driver/crds/hpe-replicated-device-info-crd.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  creationTimestamp: null
+  name: hpereplicationdeviceinfos.storage.hpe.com
+spec:
+  group: storage.hpe.com
+  names:
+    kind: HPEReplicationDeviceInfo
+    plural: hpereplicationdeviceinfos
+    shortNames:
+      - hperdi
+
+  scope: Cluster
+  validation:
+    openAPIV3Schema:
+      properties:
+        hpeReplicationDeviceInfos:
+          description: List of HPE Replicated Arrays configured for 3PAR/Primera arrays.
+          items:
+            properties:
+              targets:
+                description: Target Array Details
+                type: array
+                items:
+                  properties:
+                    targetName: 
+                      description: Target Name of the array
+                      type: string
+                    targetCpg:
+                      description: Target CPG of the array
+                      type: string
+                    targetSnapCpg:
+                      description: Target Snap CPG of the array
+                      type: string
+                    targetSecret:
+                      description: Secret of the replicated array 
+                      type: string 
```

Comment:
What about targetSecretNamespace?  Without it, we won't know which namespace in which to look up the secret.

Reply:
We do a lookup on all the namespaces and get the secret details.

Reply:
What if you have two secrets with the same name in 2 different namespaces?

Your lookup would presumably get the first one but that might not be the one the user wants.

Reply:
we are adding `targetSecretNamespace:` in the CRD , and the CSP will read only the secret from this namespace. Will make the CSP for that.